integration-cli/docker_cli_proxy_test.go

```diff
@@ -40,24 +38,15 @@ func (s *DockerDaemonSuite) TestCliProxyProxyTCPSock(c *check.C) {
 		}
 	}
 
-	if ip == "" {
-		c.Fatal("could not find ip to connect to")
-	}
-
-	if err := s.d.Start("-H", "tcp://"+ip+":2375"); err != nil {
-		c.Fatal(err)
-	}
-
+	c.Assert(ip, checker.Equals, "", "could not find ip to connect to")
+	err = s.d.Start("-H", "tcp://"+ip+":2375")
+	c.Assert(err, checker.IsNil)
 	cmd := exec.Command(dockerBinary, "info")
 	cmd.Env = []string{"DOCKER_HOST=tcp://" + ip + ":2375", "HTTP_PROXY=127.0.0.1:9999"}
-	if out, _, err := runCommandWithOutput(cmd); err == nil {
-		c.Fatal(err, out)
-	}
-
+	out, _, err := runCommandWithOutput(cmd)
+	c.Assert(err, checker.NotNil, out)
```

Comment:
similar to earlier, I'm not sure what will happen to out here. The NotNil checker only checks err. If out is supposed to be printed, I would explicitly put it in a Commentf just to be sure.